README.adoc

```diff
@@ -120,6 +120,18 @@ To enable processing of author lines as metadata, set the value of the `dita-top
 $ **asciidoctor -r dita-topic -b dita-topic -a dita-topic-authors=on _your_file_.adoc**
 ....
 
+[#callouts]
+=== Enabling callouts
+
+Unlike AsciiDoc, DITA does not support callouts as a method to add annotations to specific lines in verbatim blocks. For this reason, the conversion of callouts is disabled by default.
+
+If callouts are required, the `dita-topic` converter can use XML entities for circled numbers in place of callouts. To enable this behavior, set the value of the `dita-topic-callouts` to `on`:
```

Comment:
I think we should remove this paragraph, @jhradilek. We're asking folks to rewrite these entries rather than choose another path to callouts.

Reply:
This is more for upstream purposes, not for our case. I do not want to remove this functionality as it may be useful for somebody else converting their documentation.

Reply:
The default behavior for downstream should be the other way that is deleted in this PR. I'm okay with keeping this way in the tool.